addons/sale/views/sale_portal_templates.xml

```diff
@@ -520,7 +520,7 @@
                 <div id="total" class="row" name="total" style="page-break-inside: avoid;">
                     <div t-attf-class="#{'col-4' if report_type != 'html' else 'col-sm-7 col-md-5'} ml-auto">
                         <table class="table table-sm">
-                            <tr class="border-black" style="border-bottom:1px solid #dddddd;">
+                            <tr class="border-black" style="">
```

Comment:
Are you sure about this change? I remember I had trouble when I tried to remove it, but maybe it was on another template like the report.

Reply:
The rendering is the same visually
That border specifically will then be handled by the tax_table